src/main/resources/static/js/common.js

```diff
@@ -239,25 +239,60 @@ function viewModel(backendInstanceData) {
 // End ## Load Back end list ##
 
 // Start ## Login and Security ##
-function doIfUserLoggedIn(user) {
-    setCurrentUser(user);
+function doIfUserLoggedIn(username) {
```

Comment:
Maybe we can change this name? doIfUserLogged in says that something happens when you log in but not what. If think it should be more like "setUserInformationAndUnlockSubscriptionButton". Yes, that is a long name since this method really does a lot of things :D Atleast we will know what that method does.

Reply:
Calling multiple functions is not really good, since if one new function is added, then that function must be called from multiple places as all functions. But I agree that the naming is horrible.

setUserInformationAndUnlockSubscriptionButton() would lock this function to do only that. a better function name could be "executeAuthenticatedUserFunctions()" that wouyd indicate that in this function is several function regaridng authenticated user functionality.